### ISSUE TITLE
Caveats & Questions....

### ISSUE THREAD

**Issue by tengammaps** [dvorka/mindforger#718]
First the caveat, which is that I'm a brand new Github user and am not sure that this is the right place for what is essentially a support request; if not, I will very much appreciate being directed to the proper channels!

Question: Is there a way to completely purge a Tag from a Repository? Upon creation of a new Repo, even without Stencils or Documentation, there is a full set of Tags and I can't figure out where they're stored - its not in the repo itself (or the .mindforger.md).

If the answer is that this is not currently possible, then I'd like to submit a feature request that the Tag list for Repositories be user editable.

Thanks for this very cool tool by the way; I've just begun "playing" with it; the "paradigm shift" is substantial, but I'm excited about the possiblities!

**Comment by dvorka** [comment]
> ... not sure that this is the right place for what is essentially a support request...
Yes, this is the right way to ask question, report bug or send an enhancement request!

> Is there a way to completely purge a Tag from a Repository? 
No, it's not possible to delete a tag. There is a set of default tags (as you noticed) + custom user tags. Once you add your own tag to Notebook/Note it is added to tags and it cannot be explicitly removed. MindForger builds tag index when it starts and reads all Notebooks/Notes i.e. only custom tags which are used by a Notebook/Note are shown (after restart). Default tags are stored in source code, custom tags don't have any storage. 

> I'd like to submit a feature request that the Tag list for Repositories be user editable
If you don't mind, I would rather **hide all tags which are NOT used** (a configuration option). Would it be enough? If you add tag to a Notebook/Note, then it's like adding it. If there is no use for a tag, it will be removed.

> Thanks for this very cool tool by the way; I've just begun "playing" with it; the "paradigm shift" is substantial, but I'm excited about the possiblities!
Thank you for warm words and your interest in MindForger!

**Comment by tengammaps** [comment]
> If you don't mind, I would rather hide all tags which are NOT used (a configuration option). 

Of course! This seems like an elegant and flexible solution. Thanks so much for considering the request!

**Comment by Skrylar** [comment]
> > If you don't mind, I would rather hide all tags which are NOT used (a configuration option).
> Of course! This seems like an elegant and flexible solution. Thanks so much for considering the request!
This would be in line with how tag systems usually work in notebook apps :+1:
